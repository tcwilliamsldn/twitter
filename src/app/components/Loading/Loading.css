.loading {
    display: flex;
    width: 100vw;
    height: calc(100vh - 80px);
    justify-content: center;
    align-items: center;
    font-size: 48px;
    font-weight: bold;
    color: #757575;
    animation: fadeInOut 1s infinite;
}

@keyframes fadeInOut {
    0% {
        opacity: 1;
    }
    50% {
        opacity: 0.1;
    }
    100% {
        opacity: 1;
    }
}
