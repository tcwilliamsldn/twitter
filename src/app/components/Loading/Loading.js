import React from 'react';
import styles from './Loading.css';

const Loading = () => <div className={styles.loading}>Loading...</div>;

export default Loading;
