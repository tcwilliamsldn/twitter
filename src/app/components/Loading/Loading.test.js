import Loading from './Loading';
import React from 'react';
import { shallow } from 'enzyme';

describe('App', function() {
    it('renders the loading state', function() {
        const component = shallow(<Loading />);
        expect(component).toMatchSnapshot();
    });
});
