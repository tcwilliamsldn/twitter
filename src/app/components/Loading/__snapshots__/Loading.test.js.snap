// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders the loading state 1`] = `
<div
  className="loading"
>
  Loading...
</div>
`;
